.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--
+Thanks for opening an issue! A few things to keep in mind:
+
+- The issue tracker is only for bugs and feature requests.
+- If you need general help please post to StackOverflow http://stackoverflow.com/questions/tagged/activeadmin
+- Before reporting a bug, please search all issues, closed and open, and try reproducing your issue against
+  the latest version.
+-->
+
+### Expected behavior
+
+<!-- What do you think should happen? -->
+
+### Actual behavior
+
+<!-- What actually happens? -->
+
+### How to reproduce
+
+<!-- 
+
+Your best chance of getting this bug looked at quickly is to provide a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+See CONTRIBUTING for more details.
```

Comment:
GitHub already adds a link to CONTRIBUTING ('the guidelines') in the flash message displayed when you select 'New Issue'  I'm not sure how useful non-clickable links are when defaulted in the text input.  I guess you can click on them if you switch to 'Preview', is that the idea?